app/helpers/settings_helper.rb

```diff
@@ -0,0 +1,16 @@
+module SettingsHelper
+  def generate_form_field(setting, form)
```

Comment:
__[Reek]__

[UtilityFunction] SettingsHelper#generate_form_field doesn't depend on instance state (maybe move it to another class?)


<!-- Comment by [SideCI](https://sideci.com) -->